cliquet_docs/api/resource.rst

```diff
@@ -876,7 +876,7 @@ In a response, ``permissions`` contains the current permissions of the record
 
 .. note::
 
-    When a record is created or modified, the current :term`user id`
+    When a record is created or modified, the current :term:`user id`
     **is always added** among the ``write`` principals.
 
-`Read more about leveraging resource permissions <resource-permissions>`.
+:ref:`Read more about leveraging resource permissions <permissions>`.
```

Comment:
> I think that the way Kinto copies Cliquet docs is confusing and strange and I think it needs to be rethought, but that's outside the scope of this PR.

Definitely agree!

A year ago we didn't know that Kinto was going to be the *star* product. This needs to be cleaned-up and rethought. (Related: https://github.com/mozilla-services/cliquet/issues/687)